### PR TITLE
fix: various bugs with editor and version

### PR DESCRIPTION
**Pull request by brenkao**

### CODE REVIEW

File: client/src/components/GenerationSpans.tsx [hunk 1]
```diff
@@ -16,5 +16,6 @@ export const GenerationSpans = ({
   const { data } = useSuspenseQuery(
     spansByGenerationQueryOptions(projectUuid, generationUuid)
   );
+  console.log(data);
```
Comment (willbakst) [inline client/src/components/GenerationSpans.tsx:19]: remove?

File: lilypad/server/services/prompts.py [hunk 2]
```diff
@@ -29,6 +29,18 @@ def find_prompts_by_name(
         ).all()
         return record_tables
 
+    def get_next_version(self, project_uuid: UUID, name: str) -> int:
```
Comment (willbakst) [inline lilypad/server/services/prompts.py:32]: nit: should we put this on the BaseOrganizationSQLModel?

File: lilypad/server/api/v0/prompts_api.py [hunk 3]
```diff
@@ -203,6 +207,7 @@ def run_version(
 
 
 def _run_playground(code: str) -> str:
+    print(code)
```
Comment (willbakst) [inline lilypad/server/api/v0/prompts_api.py:210]: nit: remove

File: lilypad/generations.py [hunk 4]
```diff
@@ -73,6 +73,9 @@ def _construct_trace_attributes(
         "lilypad.generation.arg_values": json.dumps(arg_values),
         "lilypad.generation.prompt_template": prompt_template,
         "lilypad.generation.output": str(output),
+        "lilypad.generation.version": generation.version_num
+        if generation.version_num
+        else 0,
```
Comment (willbakst) [inline lilypad/generations.py:78]: does `0` mean no version number? why not `-1`?

File: lilypad/server/services/prompts.py (hunk 2 above)
Comment (brenkao) [reply lilypad/server/services/prompts.py:32]: no, there are many models without version

File: lilypad/generations.py (hunk 4 above)
Comment (brenkao) [reply lilypad/generations.py:78]: mainly for falsey but i can do -1

**Review by willbakst — APPROVED** [review]

**Pull request merged**